terraform/environments/digital-prison-reporting/main.tf

```diff
@@ -1063,6 +1126,15 @@ module "ec2_kinesis_agent" {
   account = local.account_id
   env     = local.env
 
+  ec2_sec_rules_source_sec_group = {
```

Comment:
Allows incoming traffic to the Kinesis VM with the port forwarder from the Glue Connection security group on the NOMIS port, which allows the glue job to connect to NOMIS.